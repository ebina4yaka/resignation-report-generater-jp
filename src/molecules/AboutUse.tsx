import React, { ReactElement } from 'react'
import Typography from '@material-ui/core/Typography'

export default function AboutUse(): ReactElement {
  return (
    <div>
      <Typography
        variant="h5"
        align="center"
        color="textPrimary"
        paragraph
        gutterBottom
      >
        使い方
      </Typography>
      <div>
        <Typography gutterBottom>1.各種項目を入力します。</Typography>
        <Typography gutterBottom>
          2.画面下部の「確認」ボタンを押します。
        </Typography>
        <Typography gutterBottom>
          3.表示内容を確認し、問題なければ「ダウンロード」ボタンを押します。
        </Typography>
        <Typography gutterBottom>4.完了！</Typography>
        <Typography gutterBottom>
          ※消化有給休暇日数が1日以上ある場合、ダウンロードファイルに有給取得の時季指定書が追加されます。
        </Typography>
      </div>
    </div>
  )
}
