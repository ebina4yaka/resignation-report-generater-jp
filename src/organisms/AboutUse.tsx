import React from 'react'
import Typography from '@material-ui/core/Typography'

export default function AboutUse(): React.ReactElement {
  return (
    <div>
      <Typography
        variant="h5"
        align="center"
        color="textPrimary"
        paragraph
        gutterBottom
      >
        使い方
      </Typography>
      <div>
        <Typography gutterBottom>1.各種項目を入力します。</Typography>
        <Typography gutterBottom>
          2.画面下部の「確認」ボタンを押します。
        </Typography>
        <Typography gutterBottom>
          3.表示内容を確認し、問題なければ「ダウンロード」ボタンを押します。
        </Typography>
        <Typography gutterBottom>4.完了！</Typography>
      </div>
    </div>
  )
}
