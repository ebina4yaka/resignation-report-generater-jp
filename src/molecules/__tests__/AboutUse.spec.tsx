import React from 'react'
import renderer from 'react-test-renderer'
import AboutUse from '../AboutUse'

test('Snapshot test for AboutUse', () => {
  const tree = renderer.create(<AboutUse />).toJSON()
  expect(tree).toMatchSnapshot()
})
