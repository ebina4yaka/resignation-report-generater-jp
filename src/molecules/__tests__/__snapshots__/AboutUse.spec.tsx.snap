// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test for AboutUse 1`] = `
<div>
  <p
    className="MuiTypography-root MuiTypography-h5 MuiTypography-colorTextPrimary MuiTypography-gutterBottom MuiTypography-paragraph MuiTypography-alignCenter"
  >
    使い方
  </p>
  <div>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
    >
      1.各種項目を入力します。
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
    >
      2.画面下部の「確認」ボタンを押します。
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
    >
      3.表示内容を確認し、問題なければ「ダウンロード」ボタンを押します。
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
    >
      4.完了！
    </p>
    <p
      className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
    >
      ※消化有給休暇日数が1日以上ある場合、ダウンロードファイルに有給取得の時季指定書が追加されます。
    </p>
  </div>
</div>
`;
